app/controllers/ideas_controller.rb

```diff
@@ -43,7 +43,10 @@ def create
 
   # PATCH/PUT /ideas/1
   def update
-    if @idea.update(idea_params)
+    @idea.review_year = idea_params[:review_year]
```

Comment:
These assignments shouldn't be necessary. `@idea.update(idea_params)` should add these.